nodes/scenes/cow6E6D.tmp

```diff
@@ -0,0 +1,17 @@
+[gd_scene load_steps=2 format=3 uid="uid://cnulu7wcjcw6b"]
+
+[ext_resource type="Script" path="res://nodes/ui/CowndownTimer.gd" id="1_bi2i8"]
+
+[node name="CowndownTimer" type="Node2D"]
+position = Vector2(100, 100)
```

Comment:
most notably the thing that makes me think they shouldn't be here is that I was using Vec(100, 100) for testing but though i reset it in the scene before pushing

Reply:
according to this they shouldn't be committed. https://www.reddit.com/r/godot/comments/16zmfmk/is_it_ok_to_ignore_tmp_files_from_git/

I can push an update to this pr or you can merge & remove them whoever gets to it first. We should also add to .gitignore i suppose.